roles/webots/localisation.role

```diff
@@ -0,0 +1,44 @@
+nuclear_role(
```

Comment:
is this any different to webots/robocup?

Reply:
I guess it's robocup and keyboardwalk?

Reply:
It's just keyboardwalk with localisation on, so you can move the robot around while debugging/testing localisation